protos/lobby.proto

```diff
@@ -0,0 +1,100 @@
+import "game_info.proto"
+
+package papan;
+
+message RegisterUpdate {
+  oneof update {
```

Comment:
I considered that, but it needs to be streaming anyway and streams are pretty efficient, so I think it's worth having the simplicity and having to send several messages at the beginning.